client/signup/steps/p2-confirm-email/index.jsx

```diff
@@ -75,6 +92,43 @@ function P2ConfirmEmail( { flowName, stepName, positionInFlow } ) {
 						</div>
 					</div>
 				) }
+			</>
+		);
+	};
+
+	const renderPostConfirmationNotice = () => {
+		return (
+			<>
+				<div className="p2-confirm-email__message">
+					{ translate(
+						"Thanks for confirming your email address. Your account is now active. We're almost finished!"
+					) }
+				</div>
+				<div className="p2-confirm-email__buttons">
+					<Button
+						className="p2-confirm-email__continue"
+						isPrimary={ true }
+						onClick={ handleNextStepClick }
+					>
+						{ translate( 'Continue' ) }
+					</Button>
+				</div>
+			</>
+		);
+	};
+
+	return (
+		<P2StepWrapper
+			flowName={ flowName }
+			stepName={ stepName }
+			positionInFlow={ positionInFlow }
+			headerIcon={ isEmailVerified ? check : mailIcon }
+			headerText={
+				isEmailVerified ? translate( 'Email confirmed' ) : translate( 'Check your email' )
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **28** times:
`translate( 'Email confirmed!' )` *ES Score: 10*